RELEASE_NOTES.md

```diff
@@ -1,5 +1,11 @@
 # RELEASE NOTES
 
+***********************
+# testflo version 1.4.9
+July 25, 2022
+
+- added --durations option that prints n longest running tests [#71](https://github.com/OpenMDAO/testflo/pull/71)
+
```

Comment:
I also snuck a couple of other things in there.  I fixed the way the `.testflo` config file is processed so now it should support setting any, or at least most of the testflo command line options, and because the way I did it uses type checking from `argparse`, I added the `--skip_dirs` command line option.  Before, `skip_dirs` could be defined in the config file but it wasn't a valid command line option.